src/licensedcode/data/rules/bsd-new_and_mit_2.yml

```diff
@@ -0,0 +1,4 @@
+licenses:
+    - mit
+    - bsd-new
+

```

Comment:
all the yml files have an empty line in both the `rules` and `data` (tests) folders. I thought that's a convention being followed.